src/foam/nanos/crunch/ServerCrunchService.java

```diff
@@ -373,7 +370,7 @@ public UserCapabilityJunction getJunctionForSubject(
     AuthService auth = (AuthService) x.get("auth");
     if ( auth.check(x, "service.crunchService.updateUserContext") ) {
       x = Auth.sudo(x, subject.getUser(), subject.getRealUser());
-    }
+    } else throw new AuthorizationException("You don't have permission to check for UCJs");
```

Comment:
@penzital this line breaks unit tests.
<img width="1134" alt="Screen Shot 2022-09-21 at 11 02 36 PM" src="https://user-images.githubusercontent.com/441658/191649563-1672ac0e-e8d8-49e6-b924-7db6d07c41a5.png">